pkgs/applications/audio/ymuse/default.nix

```diff
@@ -68,5 +68,7 @@ buildGoModule rec {
     description = "GTK client for Music Player Daemon (MPD)";
     license = licenses.asl20;
     maintainers = with maintainers; [ foo-dogsquared ];
+    mainProgram = "ymuse";
+    platforms = platforms.linux;
```

Comment:
Was there a regression somewhere? It builds well on Darwin before this PR.

Reply:
My bad, I just set it to `platforms.linux` since it's what I use.

Reply:
I've updated it to `platforms.unix` now.